### ISSUE TITLE
Nested multiple selector sass issue

### ISSUE THREAD

**Issue by bgriffith** [tonyganch/gonzales-pe#119]
New parse error since 3.0.0-31 with Sass and multiple selectors that are nested.

> Please check validity of the block starting from line #2

``` sass
.foo
  .bar,
  .qux
    content: 'fail'
```

Non-nested is ok

``` sass
.foo,
.bar 
  content: 'Ok'
```

**Comment by bgriffith** [comment]
hi @tonyganch - we've hit this issue again while trying to convert all our sass-lint rules to work with 3.2.1.

I'm assuming it's the same issue causing the parse error.

This time the code is:

``` sass
.foo,
.bar
  content: 'baz'

  .baz,
  .qux
    content: 'qux'
```

**Comment by tonyganch** [comment]
Should be fixed in `@3.2.5`.
